marketplace/SafeBuyFractionalStrategy.tsx

```diff
@@ -145,12 +144,10 @@ export class SafeBuyFractionalStrategy extends BuyFractionalStrategy {
         await this.exchangeClient.isTransferManagerApproved();
       // FIXME: this shouldn't be here, unless we're missing something
       if (!isTransferManagerApproved) {
-        const transferManagerApprove = await this.exchangeClient
-          .grantTransferManagerApproval()
-          .call();
-        await waitForTransactionReceipt(this.walletClient.data, {
-          hash: transferManagerApprove.hash as `0x${string}`,
-        });
+        console.debug("Approving transfer manager");
+        await this.exchangeClient.grantTransferManagerApprovalSafe(
```

Comment:
Does that mean we do need the transfer manager approval after all? If so, the FIXME above should be removed.